lib/index.js

```diff
@@ -6,26 +6,30 @@ if (typeof window !== 'undefined' && window) {
 	if (window.jQuery) {
 	  $ = window.jQuery;
 	} else {
-		$ = require('jquery')
+		$ = require('jquery');
 		window.jQuery = $;
 	}
 } else {
   $ = require('jquery');
 }
 
-var BS = require('bootstrap');
 var React = require('react');
 var objectAssign = require('object-assign');
 var getOptions = require('./get-options.js');
 var bsMultiselect = require('./bootstrap-multiselect.js');
 var bsDropdown;
-
+var BS;
```

Comment:
It looks like BS is only used within `try` - it would help readability for the variable declaration to go there.